src/services/actions/Scaffolder.ts

```diff
@@ -225,6 +264,29 @@ export class Scaffolder {
             content += ` ${ProjectFileContent.installPnPJs}`;
           }
 
+          if (newSolutionInput.shouldCreateNodeVersionFile) {
+            switch (newSolutionInput.nodeVersionManager) {
+              case 'nvm':
+                // If the node version manager is nvm, create the .nvmrc file even if the user has selected .node-version
+                content += ` ${ProjectFileContent.createNVMRCFile}`;
+                break;
+              case 'nvs':
+                // If the node version manager is nvs, create the file based on the user's settings
+                switch (newSolutionInput.nodeVersionManagerFile) {
+                  case '.nvmrc':
+                    content += ` ${ProjectFileContent.createNVMRCFile}`;
+                    break;
+                  case '.node-version':
+                    content += ` ${ProjectFileContent.createNodeVersionFile}`;
+                    break;
+                }
+                break;
+                case 'none':
```

Comment:
perfect case to be default as well. Lets add this case as `default` so that if in future we will add some new node version manager by default we will not create any file 

```suggestion
                case 'none':
                default:
```